src/app/space/forge-wizard/quickstart-wizard.component.less

```diff
@@ -0,0 +1,3 @@
+/deep/ .pfng-wizard-position-override {
+  overflow: auto;
+}
```

Comment:
The height and overflow can be set using a selector you define via the wizard config -- it should be set for both sidebar and main panel. Please see my comments here:
https://github.com/openshiftio/openshift.io/issues/1138#issuecomment-337140549

Reply:
@dlabrecq This suggestion doesn't work, because the overflow must be on the parent of the main panel. The main panel already has `overflow: auto` set.